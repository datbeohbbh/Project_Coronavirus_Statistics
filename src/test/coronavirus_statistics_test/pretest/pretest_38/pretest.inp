50 100
q 1 41 138
q 1 11 82
q 1 35 8
q 1 44 128
q 1 49 462
q 1 45 441
q 1 13 332
? 6
q 1 30 368
q 1 50 49
q 1 6 437
q 1 2 322
q 3 41 18
? 6
? 6
q 2 35 3
q 1 46 365
? 6
q 3 44 96
q 1 25 11
q 2 46 293
q 1 5 201
q 2 46 55
q 1 13 267
q 2 45 164
q 1 21 399
q 2 50 24
? 4 11
q 2 35 5
q 1 19 243
? 4 1
? 4 11
? 4 3
? 4 5
q 1 33 374
q 3 30 325
? 4 3
q 1 7 88
? 4 8
? 7
q 1 28 29
? 4 10
q 1 35 433
? 4 15
q 1 9 127
q 3 44 28
q 1 12 454
? 4 16
? 4 11
? 4 18
q 2 33 135
q 1 22 455
? 4 9
q 2 22 11
? 4 17
q 2 2 172
? 7
q 1 44 144
? 4 21
q 3 50 15
q 2 44 47
q 1 27 97
? 4 11
q 2 46 4
q 3 5 14
q 2 9 55
q 1 43 14
? 6
q 1 42 385
q 3 13 105
? 4 21
q 1 31 19
? 4 11
? 7
? 7
? 4 9
? 7
q 2 33 169
q 1 27 229
? 4 17
? 4 6
? 4 5
? 4 12
? 4 13
? 4 10
? 4 11
? 4 4
? 4 9
? 4 15
? 7
? 4 1
? 7
? 7
? 4 7
? 4 8
? 6
? 7
? 7
? 7
? 4 2