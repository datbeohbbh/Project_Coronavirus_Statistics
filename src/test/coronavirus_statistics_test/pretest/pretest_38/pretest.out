Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 3 people

Query ? 4 11 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 368
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 25 
- Information of the city 25 -
-> Number of current patients : 11
-> Has recovered : 0
-> Was dead : 0

Query ? 4 11 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 368
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 25
-> Has recovered : 24
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 11 
- Information of the city 11 -
-> Number of current patients : 82
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 25
-> Has recovered : 24
-> Was dead : 0

Query ? 4 8 :
The answer is the city with the index 41 
- Information of the city 41 -
-> Number of current patients : 120
-> Has recovered : 0
-> Was dead : 18

Query ? 7 :
The city with the highest death is city with the index 30
Quantity: 325 people

Query ? 4 10 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 201
-> Has recovered : 0
-> Was dead : 0

Query ? 4 15 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 399
-> Has recovered : 0
-> Was dead : 0

Query ? 4 16 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 399
-> Has recovered : 0
-> Was dead : 0

Query ? 4 11 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 201
-> Has recovered : 0
-> Was dead : 0

Query ? 4 18 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 437
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 41 
- Information of the city 41 -
-> Number of current patients : 120
-> Has recovered : 0
-> Was dead : 18

Query ? 4 17 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 433
-> Has recovered : 8
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 30
Quantity: 325 people

Query ? 4 21 :
The answer is the city with the index 49 
- Information of the city 49 -
-> Number of current patients : 462
-> Has recovered : 0
-> Was dead : 0

Query ? 4 11 :
The answer is the city with the index 9 
- Information of the city 9 -
-> Number of current patients : 127
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 46
Quantity: 352 people

Query ? 4 21 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 437
-> Has recovered : 0
-> Was dead : 0

Query ? 4 11 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 97
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 30
Quantity: 325 people

Query ? 7 :
The city with the highest death is city with the index 30
Quantity: 325 people

Query ? 4 9 :
The answer is the city with the index 11 
- Information of the city 11 -
-> Number of current patients : 82
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 30
Quantity: 325 people

Query ? 4 17 :
The answer is the city with the index 45 
- Information of the city 45 -
-> Number of current patients : 277
-> Has recovered : 164
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 29
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 19
-> Has recovered : 0
-> Was dead : 0

Query ? 4 12 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 101
-> Has recovered : 47
-> Was dead : 124

Query ? 4 13 :
The answer is the city with the index 41 
- Information of the city 41 -
-> Number of current patients : 120
-> Has recovered : 0
-> Was dead : 18

Query ? 4 10 :
The answer is the city with the index 11 
- Information of the city 11 -
-> Number of current patients : 82
-> Has recovered : 0
-> Was dead : 0

Query ? 4 11 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 88
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 14
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 9 
- Information of the city 9 -
-> Number of current patients : 72
-> Has recovered : 55
-> Was dead : 0

Query ? 4 15 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 187
-> Has recovered : 0
-> Was dead : 14

Query ? 7 :
The city with the highest death is city with the index 30
Quantity: 325 people

Query ? 4 1 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 10
-> Has recovered : 24
-> Was dead : 15

Query ? 7 :
The city with the highest death is city with the index 30
Quantity: 325 people

Query ? 7 :
The city with the highest death is city with the index 30
Quantity: 325 people

Query ? 4 7 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 43
-> Has recovered : 0
-> Was dead : 325

Query ? 4 8 :
The answer is the city with the index 33 
- Information of the city 33 -
-> Number of current patients : 70
-> Has recovered : 304
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 46
Quantity: 352 people

Query ? 7 :
The city with the highest death is city with the index 30
Quantity: 325 people

Query ? 7 :
The city with the highest death is city with the index 30
Quantity: 325 people

Query ? 7 :
The city with the highest death is city with the index 30
Quantity: 325 people

Query ? 4 2 :
The answer is the city with the index 25 
- Information of the city 25 -
-> Number of current patients : 11
-> Has recovered : 0
-> Was dead : 0

